driver/src/metrics/http_metrics.rs

```diff
@@ -0,0 +1,120 @@
+use anyhow::Result;
+use prometheus::{HistogramOpts, HistogramVec, Registry, DEFAULT_BUCKETS};
+use std::sync::Arc;
+use std::time::Duration;
+
+/// A registry for all HTTP related metrics.
+#[derive(Debug)]
+pub struct HttpMetrics {
+    latency: HistogramVec,
+    size: HistogramVec,
+}
+
+impl HttpMetrics {
+    /// Create a new HTTP metrics registry.
+    pub fn new(registry: &Arc<Registry>) -> Result<Self> {
+        let latency = HttpMetrics::initialize_histogram(
+            registry,
+            "dfusion_service_http_latency",
+            "Latency in seconds for HTTP request",
+            Vec::from(&DEFAULT_BUCKETS[..]),
+        )?;
+        let size = HttpMetrics::initialize_histogram(
+            registry,
+            "dfusion_service_http_size",
+            "Size in bytes for HTTP response bodies",
+            vec![
+                100.0,
+                1_000.0,
+                10_000.0,
+                100_000.0,
+                1_000_000.0,
+                10_000_000.0,
+                100_000_000.0,
+                1_000_000_000.0,
+            ],
```

Comment:
We can use https://docs.rs/prometheus/0.8.0/prometheus/fn.exponential_buckets.html for this. But I don't mind writing it out manually either. Your choice.

Reply:
Thanks! I missed so many of these `prometheus` utility functions!